tests/test_decompiler.py

```diff
@@ -44,7 +52,9 @@ def _for_all_structuring_algos(*args, **kwargs):
         ret_vals = []
         structurer_option = get_structurer_option()
         for structurer in STRUCTURER_CLASSES:
-            new_opts = orig_opts + [(structurer_option, structurer)]
+            new_opts = (
```

Comment:
this is a bad idea. Don't disable this on every single testcase like you are doing here. You have two options:
1. Disable it for _specific_ testcases and justify why
2. Fix the testcases that fail to no longer depend on the brace.